aws-provision.sh

```diff
@@ -14,44 +14,42 @@ for apt_unit in \
     sudo systemctl stop $apt_unit
     sudo systemctl mask $apt_unit
 done
-
+sudo apt-get install -y curl
 DIST=$(. /etc/os-release; echo $ID$VERSION_ID)
 VERSION_ID=$(. /etc/os-release; echo $VERSION_ID)
 
 curl -sfL https://nvidia.github.io/libnvidia-container/gpgkey | sudo apt-key add -
 curl -sfL https://nvidia.github.io/libnvidia-container/$DIST/libnvidia-container.list | sudo tee /etc/apt/sources.list.d/libnvidia-container.list
 sudo add-apt-repository -y ppa:graphics-drivers/ppa
 
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/Release.key | sudo apt-key add -
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/Release.key | sudo apt-key add -
 
 sudo apt-get update -y
 sudo apt-get install -y iproute2 ipset iptables nfs-common ssl-cert libnvidia-container-tools
-sudo apt-get install -y --no-install-recommends nvidia-cuda-toolkit nvidia-430
+sudo apt-get install -y --no-install-recommends nvidia-cuda-toolkit nvidia-driver-430
```

Comment:
Out of curiosity: why are we including nvidia drivers?

Reply:
We support gpu workloads ;)